fixed end of bridge; added fine

jrgsf opened the pull request:
This is just for Do Whatcha Wanna.

Does it maybe make sense to axe the Solo Backing entirely, since we never play it anyway?

jrgsf committed:
fixed end of bridge; added fine

michalgm reviewed (commented):
I still think we need to not call sections "A' and "B", etc - it's best that all section names clearly map to names we have hand signals for - head, bridge, hits, etc and then "1", "2", "3"

also - I need to figure out a good method to render these for PRs haha

michalgm committed:
rename sections
cleanup some directives for modern lilypond
have melody_call and melody_response parts to have multiple staffs in one melody part
clean up some repeats

michalgm committed:
Fix fine notation

michalgm committed:
remove fine from bass part

michalgm committed:
fix 'return to head' note
